liblouis/lou_translateString.c

```diff
@@ -3619,9 +3619,6 @@ translateString(const TranslationTableHeader *table, int mode, int currentPass,
 				*cursorPosition, &repwordLength, dontContract, compbrlStart, compbrlEnd,
 				beforeAttributes, &curCharDef, &groupingRule, &groupingOp);
 
-		if (transOpcode != CTO_Context)
```

Comment:
Why is this code moved further down?

Reply:
Ah, supposedly it fixes a bug in #1057.

Reply:
Ah, no it seems the other way around: #1057 mistakenly contains this PR

Reply:
A `context` rule may still be selected instead of this rule, and we don't know that yet at this point. We only know it below where I moved the code to.

`context` rules are special because they can be found by either `for_selectRule` or `findForPassRule`. It's complicated but it's the way it is.

Reply:
ah, OK, insightful, thanks